app/controllers/admin/spaces_controller.rb

```diff
@@ -0,0 +1,49 @@
+module Admin
+  # @note The ./config/routes/admin.rb file has a constraint around this controller
+  #
+  # @see https://github.com/orgs/forem/projects/46/views/1 project
+  class SpacesController < Admin::ApplicationController
+    layout "admin"
+
+    # TODO: What kind of logging do we need?  Any?  Looking for guidance.  I can assume we want to
```

Comment:
@jeremyf does belong as a comment in the code or perhaps it should a comment on the PR instead or on a project board since we're looking for guidance and suggestions?